hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/bucket/TestOMBucketDeleteRequest.java

```diff
@@ -97,6 +104,62 @@ public void testValidateAndUpdateCacheFailure() throws Exception {
         omMetadataManager);
   }
 
+  @Test
```

Comment:
For S3 it is very useful to have robot tests that tests end to end. This should be a fairly easy test to write within robot framework. Let us know if you need help with figuring out the robot test framework.

Reply:
@kerneltime Thank you for the review. Sure, I'm currently working on the robot test, will let you know when the robot test is done and the CI can be rerun.

Reply:
@kerneltime I have added the acceptance test. PTAL. Thank you.